package.json

```diff
@@ -1,17 +1,27 @@
 {
   "name": "bun-app",
   "version": "1.0.50",
+  "prisma": {
+    "seed": "node --loader ts-node/esm prisma/seed.ts"
```

Comment:
Try this
`"seed": "bun prisma/seed.ts"`
it works for me

Reply:
I don't see a seed file. Please add one. Here is an example of mine. You should use top-level await otherwise bun can face some issues

```
import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient()
console.log("🚀🚀🚀 hello ")
try {
  
  console.log('🚀🚀🚀 start ')
  const oleg = await prisma.user.create({
    data: {
      email: 'email@tut.by',
      name: 'Oleg',
      password: 'password1'
    },
  })
  console.log('🚀🚀🚀 oleg', oleg)
  
} catch (e) {
  console.log("🚀🚀🚀 error")
  console.error(e)
} finally {
  await prisma.$disconnect()
  process.exit(1)
}
```